packages/react/src/file-upload/FileUpload.stories.tsx

```diff
@@ -271,6 +271,14 @@ export const ExistingFiles: Story = {
 				name: 'police-check.pdf',
 				size: 100000,
 			},
+			{
+				name: 'police-check.pdf',
+				size: 100000,
+				href: '#',
+				onClick: () => {
```

Comment:
Should add the `event.preventDefault` here too. I'd make it consitent and use all `console.log` or all `alert`, but not a mix between the examples